### ISSUE TITLE
Insomnia TS types

### ISSUE THREAD

**Issue by mark-buhagiar** [georgeflug/insomnia-plugin-save-variables#21]
Apologies for leaving a quesion in the issues. I'm looking to write my own plugin for Insomnia, and came across your project. I noticed that you have types for Insomnia's objects in your project. Are these hand rolled, or were you able to find them somewhere else? Thanks

**Comment by georgeflug** [comment]
Hi,

They are hand-rolled because I could not find them anywhere else. I used the documentation and some poking-around to figure out the type declarations (the documentation is pretty close but missing a few things). Any definitions I have should be accurate (e.g. if the property says it's a string, then it's a string) but some of the types are incomplete with missing properties, or just a few types are missing altogether because I did not write types for things I was not using.

Feel free to copy these over to your new repo if they are a good starting point for you. Thanks!
